src/views/Contacts.vue

```diff
@@ -313,6 +322,57 @@ export default {
 			}
 		},
 
+		/**
+		 * Download group of contacts
+		 *
+		 * @param {Object} group of contacts to be downloaded
+		 * @returns {Function} trigger download of resulting single vcard file
+		 */
+		downloadGroup(group) {
+			const contacts = this.contacts
```

Comment:
Okay, a little help here :)
We already get the addressbooks from the store, so you do not need to findAllAddressBooks again.

Actually, if we pass the group object as the parametter directly, you can create an object with every addressbook and the associated contacts within:

```js
const list = {}
group.contacts.map((key) => {
	const id = this.contacts[key].addressbook.id
	list = Object.assign({
		[id]: {
			addressbook: this.contacts[key].addressbook,
			contacts: []
        }
	}, list)
	list[id].contacts.push(this.contacts[key].url)
})
```